sdcm/utils/cloud_catalog/pricing.py

```
@@ -0,0 +1,265 @@
+                else:
+                    return [price["retailPrice"] for price in prices if "Spot" in price["meterName"]][0]
+            except KeyError, IndexError:
+                LOGGER.warning("Failed to get price from prices: %s", prices)
+                return 0
+        else:
+            return 0
+
+    @staticmethod
+    @lru_cache(maxsize=None)
+    def _get_sku_prices(instance_type: str, region):
+        resp = requests.get(
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)